WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -15,12 +15,42 @@ final class DashboardViewController: UIViewController {
     private let dashboardUIFactory: DashboardUIFactory
     private var dashboardUI: DashboardUI?
 
+    private var isDefaultSiteNameAvailable: Bool {
+        return ServiceLocator.stores.sessionManager.defaultSite?.name != nil && ServiceLocator.featureFlagService.isFeatureFlagEnabled(.largeTitles)
+    }
+
+    private var titleName: String {
+        if !isDefaultSiteNameAvailable {
+            return ServiceLocator.stores.sessionManager.defaultSite?.name ?? Localization.title
+        }
+        return Localization.title
```

Comment:
I think this is a perfect use case for the `guard` pattern!
```suggestion
        guatd shouldShowStoreNameAsSubtitle {
            return Localization.title
        }
        return ServiceLocator.stores.sessionManager.defaultSite?.name ?? ""
```

Reply:
Totally agree!!